Makefile

```diff
@@ -48,16 +48,25 @@ LIB_SRC = \
 	core/parser.cpp \
 	core/static_analysis.cpp \
 	core/vm.cpp
+
 LIB_OBJ = $(LIB_SRC:.cpp=.o)
 
+LIB_CPP_SRC = \
+	$(LIB_SRC) \
+	cpp/jsonnet.cc
+
+LIB_CPP_OBJ = $(LIB_CPP_SRC:.cpp=.o)
+
 ALL = \
 	jsonnet \
 	libjsonnet.so \
+	libjsonnet-cpp.so \
```

Comment:
For some reason, running `make` only builds `jsonnet` but not the `.so` libraries, even though the `so`'s are under `ALL`.